### PR TITLE
Update to ethereumjs-vm 2.0.1

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
I don't get why this breaks the tests, a local build works for me.

**Comment by chriseth** [comment]
Tests succeed for firefox but fail for the other browsers.

File: src/app/execution-context.js [hunk 1]
```diff
@@ -16,7 +16,10 @@ if (typeof window.web3 !== 'undefined') {
   web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'))
 }
 
-var vm = new EthJSVM(null, null, { activatePrecompiles: true, enableHomestead: true })
```
Comment (yann300) [inline src/app/execution-context.js]: Why did you removed the first two parameters? I might be one the reasons why tests are failing

Comment (axic) [inline src/app/execution-context.js]: That is the breaking change of the 2.x version.

**Pull request merged**